tests/unit/test_doubles_test.sh

```diff
@@ -24,6 +24,22 @@ EOF
   assert_empty "$(assert_successful_code "$(code)")"
 }
 
+function test_successful_unmock() {
+  function code() {
+    ps | wc -l
+  }
+
+  mock ps<<EOF
+PID TTY          TIME CMD
+13525 pts/7    00:00:01 bash
+24162 pts/7    00:00:00 ps
+EOF
+
+  assert_equals "3" "$(helper::trim "$(code)")"
+  unmock ps
+  assert_not_equals "3" "$(helper::trim "$(code)")"
```

Comment:
How could return 3 on your OS?

Reply:
this has to still mock 
![image](https://github.com/TypedDevs/bashunit/assets/6353105/1afb409d-f75a-4f41-b6c0-bb40770114a8)

Reply:
an the test it's not telling you what behavior it's expected

Reply:
You're right. I will use a different approach.